Exec/RegTests/EB_ODEQty/PeleLMeX_ProblemSpecificFunctions.cpp

```diff
@@ -45,10 +45,32 @@ problem_modify_ext_sources(
 
   ParallelFor(
     *ext_src, [=] AMREX_GPU_DEVICE(int box_no, int i, int j, int k) noexcept {
-      for (int n = 0; n < NUM_ODE; n++) {
-        Real B_n = state_old_arr[box_no](i, j, k, FIRSTODE + n);
-        Real src = prob_parm.ode_srcstrength * pow(10.0, n + 1) * B_n;
-        ext_src_arr[box_no](i, j, k, FIRSTODE + n) += src;
+      if (prob_parm.ode_qty_test) {
+        for (int n = 0; n < NUM_ODE; n++) {
+          // Source terms for ODE qty test
+
+          Real B_n = state_old_arr[box_no](i, j, k, FIRSTODE + n);
+          Real src = prob_parm.ode_srcstrength * pow(10.0, n + 1) * B_n;
+          ext_src_arr[box_no](i, j, k, FIRSTODE + n) += src;
+        }
+        // Ignore time as it is only used if composition_test = 1
+        amrex::ignore_unused(time);
```

Comment:
This ignore unused and the one below shouldn't be needed. ignore-Unused should only be needed when certain variables are excluded by compile time macros, not runtime conditionals

Reply:
Thanks for clarifying.  I made the update and merged the most recent branch.